mentor_classifier/mentor_classifier/__init__.py

```diff
@@ -78,6 +78,7 @@ def register_classifier_factory(arch: str, fac: ArchClassifierFactory) -> None:
 
 
 ARCH_LR = "mentor_classifier.arch.lr"
+ARCH_LR_TRANSFORMER = "mentor_classifier.arch.transformers"
```

Comment:
let's put `lr` as a prefix to the arch name too, e.g. `mentor_classifier.arch.lr_transformers`